src/main/java/hudson/plugins/ec2/SpotConfiguration.java

```diff
@@ -3,29 +3,29 @@
 import org.kohsuke.stapler.DataBoundConstructor;
 
 public final class SpotConfiguration {
+    public final Boolean useBidPrice;
```

Comment:
Can you make this a `boolean` instead? If `boolean` doesn't work properly with `@DataBoundConstructor`, you can still use `Boolean` in the constructor args and unbox it there (convert `null` to whatever the default setting should be).